### PR TITLE
add search bars

### PR DESCRIPTION
**Pull request by manch0ffline**
- [DEMO LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
```
Comment (volodymyr-soltys97) [inline src/index.html:14]: Use a semantic tag for for inputs instead of main

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,33 @@
...
+      <label for="input-big" class="main__label main__label--big">
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="img img--big">
```
Comment (volodymyr-soltys97) [inline src/index.html:19]:
```suggestion
        <img
          src="images/Search.svg"
          alt="search"
          class="img img--big"
        >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,33 @@
...
+          placeholder="Try &#34;Los Angeles&#34;"
+          id="input-big"
+          data-qa="big"
+        >
+      </label>
+      <label for="input-small" class="main__label main__label--small">
+        <img
+          src="images/Search.svg"
+          alt="search"
+          class="img img--small">
```
Comment (volodymyr-soltys97) [inline src/index.html:32]:
```suggestion
        <img
          src="images/Search.svg"
          alt="search"
          class="img img--small"
        >
```

File: src/index.html (hunk 2 above)
Comment (volodymyr-soltys97) [inline src/index.html:19]: Checklist, you need to use image as a background property
image

File: src/style.css [hunk 4]
```diff
@@ -1 +1,111 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc)
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/style.css:9]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,111 @@
...
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --border-bars: #E1E7ED;
+  --bg-page: #FFF;
+  --shadow-bars: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  --shadod-bars-hover: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+  --text-color: #3D4E61;
```
Comment (volodymyr-soltys97) [inline src/style.css:27]: Create a variable with root selector

File: src/style.css [hunk 6]
```diff
@@ -1 +1,111 @@
...
+
+  padding: 0;
+  font-family: Avenir, sans-serif;
+}
+
+/* #region lables */
```
Comment (volodymyr-soltys97) [inline src/style.css:33]: Remove comments
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/style.css [hunk 7]
```diff
@@ -1 +1,111 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc)
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
```
Comment (maxim2310) [inline src/style.css:9]: also need to add font weight

File: src/style.css (hunk 5 above)
Comment (maxim2310) [reply src/style.css:26]: not fixed https://github.com/mate-academy/layout_search-bar-airbnb/pull/3817/files#r1407405161

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Pull request closed without merging**